src/labkey/filter/Types.ts

```diff
@@ -543,7 +545,8 @@ export function registerFilterType(
                     if (value.indexOf('{json:') === 0 && value.indexOf('}') === value.length - 1) {
                         value = JSON.parse(value.substring('{json:'.length, value.length - 1));
                     } else {
-                        value = value.split(type.getMultiValueSeparator());
+                        const regexPattern = new RegExp(`[${getSecondaryMultiValueSeparator()}${type.getMultiValueSeparator()}]`);
```

Comment:
Any reason getSecondaryMultiValueSeparator can't just be a const instead of function?

const NEW_LINE_SEP = '\n';
const regexPattern = new RegExp(`[NEW_LINE_SEP${type.getMultiValueSeparator()}]`);